### ISSUE TITLE
Appstore not works

### ISSUE THREAD

**Issue by RuesanG** [Sticktron/g0blin#249]
Hello  im connected my phone with my home wifi and my appstore download works 

and im in the city and connected with a open wifi and the downloads not works and from open wifi to my home wifi the downloads not works im make a complete reboot and rejailbreaking and the downloads works can you help me im delete all tweaks and cleaned with icleaner this problem its not solved its a bug from g0blin ? When yes can you soon fix this ? 

English is bad 😅

**Comment by ringosham** [comment]
I can confirm there is a problem with App Store downloads.
But rebooting (Not respring) the device will make it work again.
Could it be related to this? #241

**Comment by RuesanG** [comment]
yes can you help us inexperienced somehow help what it could be?

**Comment by RuesanG** [comment]
can close this ? or open ?

**Comment by RuesanG** [comment]
Video to the Problem Here: part 1 https://www.youtube.com/watch?v=TfXNr29G5JU&feature=youtu.be

part 2 to the problem : https://www.youtube.com/watch?v=5ae0a9zRBAA&feature=youtu.be

**Comment by RuesanG** [comment]
https://github.com/tihmstar/doubleH3lix/issues/5

Problem its founded
